blocks/article-body-block/chains/article-body/default.test.jsx

```diff
@@ -1468,6 +1468,286 @@ describe('article-body chain', () => {
+                },
+                height: 3744,
+                image_type: 'photograph',
+                last_updated_date: '2019-07-09T22:29:42Z',
+                licensable: false,
+                owner: {
+                  id: 'corecomponents',
+                  sponsored: false,
+                },
+                source: {
+                  name: 'Death to Stock Photo',
+                  source_type: 'stock',
+                  edit_url: 'https://corecomponents.arcpublishing.com/photo/CITIAYX2ERDOPP2TPJGEUV7SNQ',
+                  system: 'Anglerfish',
+                },
+                subtitle: 'Australia surf trip',
+                taxonomy: {
+                  associated_tasks: [
+
+                  ],
+                },
+                type: 'image',
+                url: 'https://arc-anglerfish-arc2-prod-corecomponents.s3.amazonaws.com/public/CITIAYX2ERDOPP2TPJGEUV7SNQ.jpg',
+                version: '0.9.0',
+                width: 5616,
+                resized_params: { '1440x0': '' },
+              },
+            ],
+          },
+          arcSite: 'the-sun',
+          customFields: { },
+        })),
+      }));
+      const { default: ArticleBodyChain } = require('./default');
+      const wrapper = mount(
+        <ArticleBodyChain>
+          <div>1</div>
+          <div>2</div>
+          <span>3</span>
+        </ArticleBodyChain>,
+      );
+      expect(wrapper.find('figure').length).toEqual(1);
+      expect(wrapper.find('figure').find('Image').length).toEqual(1);
+      expect(wrapper.find('figure').find('figcaption').length).toEqual(1);
+      expect(wrapper.find('figure').find('figcaption').find('span').length).toEqual(1);
+      expect(wrapper.find('figure').find('figcaption').find('p').length).toEqual(1);
+      expect(wrapper.find('figure').find('figcaption').find('p').find('.title').length).toEqual(1);
+      expect(wrapper.find('figure').find('figcaption').find('p').text()).toMatch("Australia surf trip Australia's great ocean road is home to the kind of stuff you see in magazines and always wish you could visit one day: Twelve Apostles, Koalas, Kangaroos, surf towns, Bells Beach and Point Break.");
+      expect(wrapper.find('figure').find('figcaption').find('p').find('.title')
+        .text()).toMatch('Australia surf trip ');
+    });
+
+    it('should override photographer and credit by using vanity_credits', () => {
+      jest.mock('fusion:properties', () => (jest.fn(() => ({
+        resizerURL: 'https://fake.cdn.com/resizer',
+      }))));
+
+      jest.mock('fusion:context', () => ({
+        useAppContext: jest.fn(() => ({
+          globalContent: {
+            content_elements: [
+              {
+                caption: 'my cool caption',
+                subtitle: 'my cool subtitle',
+              },
+            ],
+          },
+        })),
+        useFusionContext: jest.fn(() => ({
+          globalContent: {
```

Comment:
More like a nitpick but we could probably trim the unused parts of this context object so the file doesn't get too long to navigate